package.json

```diff
@@ -19,15 +19,19 @@
     "gulp": "^4.0.0",
     "gulp-autoprefixer": "^6.0.0",
     "gulp-eslint": "^5.0.0",
+    "gulp-iconfont": "^10.0.1",
+    "gulp-iconfont-css": "^2.3.0",
     "gulp-if": "^2.0.2",
     "gulp-rename": "^1.4.0",
     "gulp-replace": "^1.0.0",
     "gulp-sass": "^4.0.2",
     "gulp-sass-lint": "^1.4.0",
     "gulp-size": "^3.0.0",
     "gulp-sourcemaps": "^2.6.4",
+    "gulp-svg-sprite": "^1.5.0",
     "inquirer": "^6.2.1",
     "node-sass-magic-importer": "^5.2.0",
+    "typey": "^1.0.3",
```

Comment:
i think typey adding should be disqussed.

Reply:
what's the problem with is? I'd say we use it a lot https://github.com/jptaranto/typey

Reply:
what to use instead? 
TODO was added here for now https://github.com/skilld-labs/kaizen/pull/22/files#diff-2f9ab01abf385e71eb46e4f91b3d36b2R2

Reply:
Typey is a good library and we use it in the all project.

Reply:
I did research about type alternatives and didn't find anything better. We all know how type is working so maybe no sense to change replace it?